### PR TITLE
Esper Device SDK Documentation Update

### PR DESCRIPTION
**Pull request by TechJorvis**
**Updated**:
- organised table of contents
- sorted the SDK method headings alphabetically
- SDK current version
- SDK release name

**Added**:
- method documentation for:
   - getDeviceTemperatures
   - pushTelemetryData
- method implementation for:
   - `startDock` method under `Manage Dock` heading

**Corrected**:
- typo in `setSystemSetting` method name
- method tag used in `setSystemSetting` method

### CODE REVIEW

File: docs/home/devicesdk.md [hunk 1]
```diff
@@ -907,3 +930,73 @@ sdk.getEsperRemovableStorageCachePath(new EsperDeviceSDK.Callback<String>() {
...
+ * @param callback - callback implementation to be invoked upon completion of the operation.
+ *                 In devices running android 7 or above, the callback will return float array
+ *                 of temperatures of given type and source.
+ *                 Callback's onFailure method returns InvalidAndroidSdkException in case Device is
+ *                 running below Android 7
+ *                 Callback's onFailure method returns InternalError in case device
+ *                 is not able to access {@link android.os.HardwarePropertiesManager}
+ *
+ */
+sdk.getDeviceTemperatures(0, 0, new EsperDeviceSDK.Callback<float[]>() {
```
Comment (mhrpatel12) [inline docs/home/devicesdk.md:962]: can we put constant here instead of 0, 0?

Comment (TechJorvis) [reply docs/home/devicesdk.md:962]: yeah makes sense... 👍

**Review by mhrpatel12 — APPROVED** [review]

File: docs/home/devicesdk.md [hunk 2]
```diff
@@ -118,7 +126,213 @@ sdk.isActivated(new EsperDeviceSDK.Callback<Boolean>() {
...
+    @Override
+    public void onFailure(Throwable t) {
+
+    }
+});
+```
+
+The API allows you to pass a list of packages whose data is to be cleared. The API will return a null in onResponse if it was successfully able to clear the data of all apps or it shall return the list of packages whose data it was unable to clear.
+
+### Configure APNs via the Esper Device SDK
```
Comment (mhrpatel12) [inline docs/home/devicesdk.md:211]: why is this json placed in between `clearAppData` and `configNoNetworkFallback` ?

Comment (TechJorvis) [reply docs/home/devicesdk.md:211]: umm.. not sure... as far as I remember I didn't added this 😅 ... I just simply sorted the sections in alphabetically ascending order.

I think it was given as a sample example.

**Review by mhrpatel12 — APPROVED** [review]

**Pull request merged**